R/addOrderSequence.R

```diff
@@ -0,0 +1,57 @@
+#' Rank the sample information in a `SummarizedExperiment` Object
```

Comment:
As far as I can see, an equivalent of this function can be implemented with just one line:

`colData(hitchip1006)$rank <- rank(colData(hitchip1006)$time)`

This would be more transparent, flexible, and does not require maintenance. 

Hence I suggest to remove `addOrderSequence` from this PR.

However, we may like to do a subject-specific version of this for a new PR. Let us rethink that more properly in a next PR after merging the current one. Remove this function from the current PR.